### ISSUE TITLE
Add cyrillic symbols

### ISSUE THREAD

**Issue by Ygarr** [ninjaaron/bitocra#5]
Showing cyrillic symbols is big, blurred, shifted and broken - please fix

**Comment by hexvalid** [comment]
this font not contains any cyrilic symbols. its going to fallback. i preffer use pretty good bdf fallback font

**Comment by ninjaaron** [comment]
I have to agree with @hexvalid. Use a Cyrillic fallback font for now. There is no Cyrillic script in these fonts because I don't read Cyrillic and don't trust my intuition about what good shapes for the letters would be that work with the current visual style of bitocra.

However, I will leave this issue open because I'd be very happy to work with someone more familiar with the writing system to come up with an acceptable set of glyphs, so I can better serve users of this script.
